translate/rus.json

```diff
@@ -332,7 +332,7 @@
 "reset":"сброс",
 "resolution":"разрешение",
 "result":"результат",
-"reverse kills":"обратные убийства",
+"reverse kills":"двойные убийства",
```

Comment:
Yeah, I think I found the way to properly show 'double wins' in a way that it is both stylish and explanative. I think you can use some of the 'slang-ish' words in the translation, but also use a pop-up window to explain what does this 'slang' actually mean. What do you think? Okay or off?